### PR TITLE
Display External Datasets Within Subcategory

### PR DESCRIPTION
**Pull request by isaaccodekill**
image

### CODE REVIEW

File: app/src/app/api/v0/inventory/[inventory]/value/subsector/[subsector]/route.ts [hunk 1]
```diff
@@ -13,7 +13,15 @@ export const GET = apiHandler(async (_req, { params, session }) => {
...
-    include: [{ model: db.models.DataSource, as: "dataSource" }],
+    include: [
+      { model: db.models.DataSource, as: "dataSource" },
+      {
+        model: db.models.SubCategory,
+        as: "subCategory",
+      },
+      { model: db.models.Sector, as: "sector" },
+      { model: db.models.SubSector, as: "subSector" },
+    ],
```
Comment (lemilonkh) [inline app/src/app/api/v0/inventory/[inventory]/value/subsector/[subsector]/route.ts:24]: Can you add this to the documentation of the endpoint in the Github wiki API docs?

File: app/src/components/Tabs/Activity/external-data-section.tsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    api.useDisconnectThirdPartyDataMutation();
+
+  const {
+    isOpen: isSourceDrawerOpen,
+    onClose: onSourceDrawerClose,
+    onOpen: onSourceDrawerOpen,
+  } = useDisclosure();
+  const onSourceClick = (source: DataSourceWithRelations, data: any) => {
+    // setSelectedSource(source);
+    // setSelectedSourceData(data);
```
Comment (lemilonkh) [inline app/src/components/Tabs/Activity/external-data-section.tsx:49]: Can we remove the commented code or is it crucial to have?

**Review by lemilonkh — APPROVED** [review]
Thanks :sparkles:

File: app/src/app/api/v0/datasource/[inventoryId]/route.ts [hunk 3]
```diff
@@ -138,6 +138,8 @@ export const GET = apiHandler(async (_req: NextRequest, { params }) => {
   const sources = sectorSources
     .concat(subSectorSources)
     .concat(subCategorySources);
+
+  console.log("sources", sources);
```
[the remark this answers is not in the document]
Comment (isaaccodekill) [reply app/src/app/api/v0/datasource/[inventoryId]/route.ts:142]: will do

**Pull request merged**